src/ripple/rpc/handlers/Submit.cpp

```diff
@@ -48,6 +48,20 @@ doSubmit(RPC::JsonContext& context)
 {
     context.loadType = Resource::feeMediumBurdenRPC;
 
+    RPC::SubmitSync sync = RPC::SubmitSync::sync;
+    if (context.params.isMember(jss::sync))
+    {
+        if (context.params["sync"].asString() == "async")
+            sync = RPC::SubmitSync::async;
+        else if (context.params["sync"].asString() == "wait")
+            sync = RPC::SubmitSync::wait;
+        else if (context.params["sync"].asString() != "sync")
```

Comment:
I don't like using the string value instead of the `jss` to get the param, but in addition, it's being done 3 times. How about:
```
        auto const syncParam = context.params[jss::sync].asString();
        if (syncParam == "async")
            sync = RPC::SubmitSync::async;
        else if (syncParam == "wait")
            sync = RPC::SubmitSync::wait;
        else if (syncParam != "sync")
```

Reply:
fixed